examples/petsc/qfunctions/area/areasphere.okl

```diff
@@ -132,11 +132,11 @@ typedef double CeedScalar;
 // This QFunction applies the mass matrix for a scalar field.
 //
 // Inputs:
-//   u     - Input vector at quadrature points
+//   in     - Input vector at quadrature points
 //   qdata - Geometric factors
 //
 // Output:
-//   v     - Output vector (test function) at quadrature points
+//   out     - Output vector (test function) at quadrature points
```

Comment:
Listing `in` and `qdata` as inputs doesn't make sense. `in` is all input arrays packed into a single array, containing the arrays `u` and `qdata`. `u` starts at the location `in + iOf7[0]` and `qdata` at `in + iOf7[1]`.

I'd stay with `u` and `v`, personally